nixos/doc/manual/configuration/gpu-accel.chapter.md

```diff
@@ -64,15 +64,15 @@ enables OpenCL support:
 
 ### Intel {#sec-gpu-accel-opencl-intel}
 
-[Intel Gen8 and later
-GPUs](https://en.wikipedia.org/wiki/List_of_Intel_graphics_processing_units#Gen8)
+[Intel Gen12 and later GPUs](https://en.wikipedia.org/wiki/List_of_Intel_graphics_processing_units#Gen12)
 are supported by the Intel NEO OpenCL runtime that is provided by the
-intel-compute-runtime package. The proprietary Intel OpenCL runtime, in
-the intel-ocl package, is an alternative for Gen7 GPUs.
+intel-compute-runtime package. The previous generations (8,9 and 11), have been moved to the
+intel-compute-runtime-legacy1 package by Intel.
+The proprietary Intel OpenCL runtime, in the intel-ocl package, is an alternative for Gen7 GPUs.
```

Comment:
```suggestion
The previous generations (8,9 and 11), have been moved to the intel-compute-runtime-legacy1 package.
The proprietary Intel OpenCL runtime, in the intel-ocl package, is an alternative for Gen7 GPUs.
```
That package is not really from Intel.
Also one sentence per line is easy to review and read in code